scripts/model-writer.js

```diff
@@ -17,6 +17,29 @@ $(document).ready(function(){
     });
 });
 var ModelWriter = {};
+
+ModelWriter.formGetter = function()
+{
+    var form = $("form#form-model-writer");
+    var returnObject = {};
+    returnObject.language = form.find("#language").val();
+    returnObject.company = form.find("#company").val();
+    returnObject.namespace = form.find("#namespace").val();
+    returnObject.model = form.find("#model").val();
+    returnObject.args = [];
+    var args = $(".form-group.args");
+    var names = $("[name='arg_name[]'");
+    var types = $("[name='arg_type[]'");
+    console.log(args);
```

Comment:
Please remove the console.log(args), once you were done testing, it was no longr needed. Also leave this as a blank line.